notebooks/p-center.ipynb

```diff
@@ -68,7 +67,7 @@
   },
   {
```

Comment:
The `MAX_COVERAGE`/`service_radius` parameters are only applicable for the MCLP and LSCP. For the *p*-median and *p*-center problems the only parameter should be your `P_FACILITIES` parameter. I noticed this while reviewing the results and saw that 5/5 candidate facility sites were being selected, when it should be 4/5. We'll need to update the code cells (23 & 26) and explanations before them.

* check the plot titles for the final results in cell 25 😅 and rerun the final ***PCenter from geodataframes*** cell.
* review the *p*-median notebook for similar inconsistencies.


Things are looking good here. These are simply minor clean up & revision activities that happen in any PR this size.